### ISSUE TITLE
Question about changing the slider value regardless of the handler position or where the user touch the track

### ISSUE THREAD

**Issue by 32mandev** [Ali-Azmoud/flutter_xlider#71]
Hello and thanks for this very nice and helpful library.

I would like to have a slider which, when the user touch the track and move his/her finger for example 20 pixels to the right, increase the value in proportion (until we reach the maximum), regardless of the handler position.

On a default slider, when the finger touch the track and move, the handler jump to the finger position, even if the jump parameter is set to false.

Is there a way to get this behavior?

**Comment by Ali-Azmoud** [comment]
Hi.
I think you should set `selectByTap` to false. Let me know if thats not what you want

**Comment by 32mandev** [comment]
Hi,
I already tried `selectByTap : false, jump : false` but it's doesn't give this behaviour.

Let me try to explain it. Suppose you have a slider like that:

```
-100                      0                          100                  
                          --------------->
     ^User press here
```
Let's say the current value is 50.

When the user press his/her finger around -75 and move it one centimeter to the right, currently the handler jump to the finger position and then the value change to something like -55 = -75+20.
I'd like the value change to 50+20=70.

In other words, I would like to have the value increased by 20 when the user move his/her finger one centimeter to the right, regardless of where the finger starts.

**Comment by Ali-Azmoud** [comment]
I think you should use `onDragging`. you can define a custom variable and do your calculations based on lower and upper values and put the result in your custom variable and display it on the screen.

**Comment by 32mandev** [comment]
Thanks!
